KK-761 | Add support for Ticketmaster ticketing system

Jokinen opened the pull request:
## Description

Adds support for Ticketmaster ticketing system.

Events which have been configured with the Ticketmaster ticketing system will use the redirect flow for reservations.

This means that users will not sign up for the event within kukkuu. Instead, they will be notified that they will be redirected to the Ticketmaster service.

## Context

<!-- Why is this change required? What problem does it solve? -->
<!-- Leave a link to the Jira ticket for posterity. -->

[KK-761](https://helsinkisolutionoffice.atlassian.net/browse/KK-761)

tuovinensanttu commented on src/domain/event/EventOccurrenceRedirect.tsx:
```diff
@@ -0,0 +1,140 @@
+            <IconCheck className={styles.successCheckMark} />
+          )}
+        </div>
+        <div className={styles.row}>
+          <LinkButton
+            variant="secondary"
+            to={getEventPathname(location.pathname)}
+            className={styles.linkButton}
+          >
+            {t('eventOccurrenceRedirectPage.back')}
+          </LinkButton>
+          <AnchorButton href={ticketmasterUrl}>
+            {t('eventOccurrenceRedirectPage.continue')}
+          </AnchorButton>
+        </div>
```
On smaller screen these buttons being on the same line feels little cramped. I haven't seen the designs for this, but in my opinion it would be better if they were under each other and had full width.

Jokinen replied:
There was no mobile version of the design. I'll make this change.

tuovinensanttu reviewed (approved):
LGTM 👍

tuomas777 reviewed (approved):
Very great!! 💯 

A couple of details:

- we probably don't want to show the number of free places of Ticketmaster events. The design also has it that way, but that change is unfortunately missing from the ticket
- the past event text "Please take into consideration that tickets which have been delivered through Ticketmaster..." should not be always visible. There are currently no clear specs about that, most probably it should be shown only for children of certain projects (years). Of course this can be merged now and conditional hiding added later, we just need to make sure we remember to do that 🙂

Jokinen commented on src/common/components/text/Text.tsx:
```diff
@@ -0,0 +1,53 @@
+    case 'body':
+    case 'body-l':
+    default:
+      return 'p';
```
Codewise they don't do much, but for humans they tell that those two variants are accounted for.

Jokinen commented:
I'll remove the ticket count column.

Personally I'm not sure if we can't just hardcode the text 🤔 It doesn't seem too harmful. But yeah, it's something we can't show conditionally even if we wanted to so it can't be resolved now.

Jokinen commented:
Thank you for the comments! I adjusted the mobile layout and dropped the unnecessary ticket count

<img width="994" alt="Screenshot 2021-07-05 at 10 55 32" src="https://user-images.githubusercontent.com/9090689/124437315-9d070480-dd7f-11eb-8063-eb8f16a57f92.png">
<img width="378" alt="Screenshot 2021-07-05 at 10 55 16" src="https://user-images.githubusercontent.com/9090689/124437323-9ed0c800-dd7f-11eb-905e-e590ab9ba3a3.png">
